.readthedocs.yml

```diff
@@ -16,4 +16,4 @@ sphinx:
 # Optionally set the version of Python and requirements required to build your docs
 python:
   install:
-    - requirements: doc-requirements.txt
+    - requirements: doc-requirements.in
```

Comment:
I think using the pinned version `.txt` is safer for doc builds. Otherwise, we can have PRs that break from unrelated issues.

Reply:
Currently, I do not see `tensorflow-macos` in `doc-requirements.txt`. Is this change still required?

> planning on using conda here

Yes, `Conda-lock` will resolve this because it can lock a linux environment from a macos environment.

In any case, I am okay with merging this as is and fixing it later if doc builds become flaky.